src/main/java/CommandCenter.java

```diff
@@ -0,0 +1,14 @@
+import java.util.ArrayList;
+
+public class CommandCenter {
```

Comment:
Possibly remove the class if it is not used, unless it will be used in the future developments?